lib/aff4.py

```diff
@@ -46,6 +46,8 @@ class Error(Exception):
 class LockError(Error):
   pass
 
+class OversizedRead(Error):
```

Comment:
this should extend IOError so it can be caught by IOError (as it is normal for read() to possibly fail by IOError).